src/index.js

```diff
@@ -1,2 +1,172 @@
+    });
+  });
+
+  const removePng = document.querySelectorAll('.delete');
+  removePng.forEach((i) => {
+    i.addEventListener('click', () => {
+      removetodo(i.parentElement);
+    });
+  });
+  localStorage.setItem('activities', JSON.stringify(Array1));
+};
+
+const edittodo = (todocontainer, todo) => {
+  const editinput = document.createElement('input');
+  editinput.type = 'text';
+  editinput.className = 'edittext';
+  editinput.value = todo.textContent;
+  todocontainer.replaceChild(editinput, todo);
+  editinput.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      const todocontainers = document.querySelectorAll('.todocontainer');
+      const localData = JSON.parse(localStorage.getItem('activities'));
+      for (let i = 0; i < todocontainers.length; i++) {
+        if (todocontainers[i].classList.contains('checkedcontainer')) {
+          localData[i].description = editinput.value;
+          localStorage.setItem('activities', JSON.stringify(localData));
+        }
+      }
+      editinput.parentElement.classList.remove('checkedcontainer');
+      todocontainer.replaceChild(todo, editinput);
+      todo.textContent = editinput.value;
+    }
+  });
+};
+
+const removetodo = (todo) => {
+  todoCon.removeChild(todo);
+  let count = 0;
+  const localData = JSON.parse(localStorage.getItem('activities'));
+  const data = Array.from(localData).filter((i) => i.completed === false);
+  data.map((i) => i.index = count += 1);
+  localStorage.setItem('activities', JSON.stringify(data));
+};
+
+textInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && textInput.value) {
+    addToList(textInput.value);
+    textInput.value = null;
+  }
+});
+
+const getFromLocal = () => {
+  const data = JSON.parse(localStorage.getItem('activities'));
+  data.map((i) => {
+    Array1.push(i);
+    const todocontainer = document.createElement('div');
+    todocontainer.className = 'todocontainer';
+    todocontainer.innerHTML += `
+      <input type="checkbox" class="check">
+      <span>${i.description}</span>
+      <i class="fa-solid fa-ellipsis-vertical menu"></i>
+      <i class="fa-solid fa-trash-can delete"></i>
+    `;
+    todoCon.appendChild(todocontainer);
+
+    const edit = document.querySelectorAll('.menu');
+    edit.forEach((i) => {
+      i.addEventListener('click', () => {
+        edittodo(todocontainer, i.previousElementSibling);
+        i.parentElement.classList.add('checkedcontainer');
+      });
+    });
+  });
+
+  const checkbox = document.querySelectorAll('.check');
+  checkbox.forEach((i) => {
+    i.addEventListener('click', () => {
+      i.parentElement.classList.toggle('checkedcontainer');
+      i.nextElementSibling.classList.toggle('inputvalue');
+      i.parentElement.lastElementChild.classList.toggle('trash');
+      i.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-disable');
+      updateLocal();
+    });
```

Comment:
You have the same block of code in lines 26 to 33. Following the DRY principle, you should not be repeating blocks of codes